server/channels/app/team_test.go

```diff
@@ -635,7 +673,8 @@ func TestPermanentDeleteTeam(t *testing.T) {
 	require.Nil(t, err, "Should create a team")
 
 	defer func() {
-		th.App.PermanentDeleteTeam(th.Context, team)
+		appErr := th.App.PermanentDeleteTeam(th.Context, team)
+		require.Nil(t, appErr)
```

Comment:
Hey @hanzei  having an issue with this particular test, fixed all the errcheck issues, but the test is failing. Using require.Nil() results in an error, since an error is expected, but switching it to NotNil results in this error: runtime error: 'invalid memory address or nil pointer dereference', not sure how to proceed to clear this.

Reply:
The test is just broken. In L719, it deleted the team, only to delete it again in the `defer` function. Can you please cleanup the unnecessary calls to `th.App.PermanentDeleteTeam`?